scenarios/helpers/iq_metrics.erl

```diff
@@ -0,0 +1,108 @@
+-module(iq_metrics).
+
+-export([start/1, request/2, request/3, timeout/2, response/2, new_id/1]).
+
+-record(request, {
+    key :: {Id :: binary(), pid()},
+    name :: name(),
+    time :: integer(),
+    tref :: timer:tref() | undefined | timed_out
+}).
+
+-type name() :: atom().
+
+-define(TABLE, request).
+
+-spec start([name()]) -> ok.
+start(Names) ->
+    {ok, App} = application:get_application(),
+    PrivDir = code:priv_dir(App),
+    application:set_env(mnesia, dir, PrivDir),
+
+    mnesia:create_schema([node()]),
+    application:ensure_started(mnesia),
+    mnesia:create_table(?TABLE, [{attributes, record_info(fields, request)}]),
```

Comment:
As far as I understand we want mnesia here in order to have the information replicated to all amoc nodes. Am I correct? If so did you check if table created in this way is in fact replicated to all the nodes?

Reply:
Also I think this table can be ram only. Did you check if this configuration gives us this behaviour?

Reply:
When ran locally with this configuration it is ram only but perhaps it should be ensured explicitly

Reply:
I think writing to the table can be changed to `dirty` instead of transaction.